test/mockPaymentRequest.js

```diff
@@ -0,0 +1,33 @@
+const paymentRequest = {
```

Comment:
suggest file is renamed `mock-payment-request.js` to be consistent.  Rather than camels.